elements/urth-viz-bar/urth-viz-bar.html

```diff
@@ -31,6 +31,17 @@
     Polymer({
         is: 'urth-viz-bar',
 
+        properties: {
+            /**
+             * Allows user to select multiple bars.
+             */
+            multiSelect: {
```

Comment:
Instead of placing the `multiSelect` property in each chart type, I think the best approach is to place it on `elements/urth-viz-chart/urth-viz-chart-common-behavior.html`.